src/commands/lab.rs

```diff
@@ -102,19 +100,101 @@ async fn handle_vscode_integration(
 ) -> Result<()> {
     is_vscode_available(pb).await?;
 
-    install_extensions(pb).await?;
-    create_vscode_settings(&global_args.project, env)?;
+    if VSCodeSettings::load()
+        .await?
+        .aqora_can_install_extensions()
+        .unwrap_or(false)
+    {
+        install_extensions(pb).await?;
+    }
 
+    create_vscode_settings(&global_args.project, env)?;
     open_vscode(global_args.project, pb).await?;
+    Ok(())
+}
 
+async fn ask_for_install_vscode_extensions(
+    allow_vscode_extensions: Option<bool>,
+    pb: &ProgressBar,
+) -> Result<()> {
+    let mut vscode_settings = VSCodeSettings::load().await?;
```

Comment:
I would store this setting not on the project basis but on the user basis because installing extensions is not on a project basis but user basis, so if they so, "no I don't want to install these extensions", they shouldn't have to be prompted every time they add a new aqora template. So I would put it into `config_dir`

Reply:
Oh! I think the struct name was unclear -sorry! I'm storing these settings in the user settings file, not the project one ;)

Reply:
my bad, i saw some changes to the vscode settings in the project dir and just kind of assumed but didn't actually read it carefully